solutions/usaco-991.mdx

```diff
@@ -68,4 +68,44 @@ public class Loan {
   
 </JavaSection>
   
+<PySection>
+```
+# our checking function in the binary search
+def isOk(numGallons, withinDays, atLeast, checkNumber):
```

Comment:
1. use snake_case
2. type annotations
3. `isOk` tells me nothing about what the function actually does

Reply:
point 3 isn't addressed

Reply:
don't assume the reader knows what "checking" means
what are we checking? what are the parameters? what does it return?
something like "checks whether FJ can [do something] given [stuff]"

Reply:
Yeah I've updated and raised a new merge request. please review it once

Reply:
@SansPapyrus683 Could you please review my updated changes

Reply:
I've added a function description, in the new merge please check out

https://github.com/cpinitiative/usaco-guide/pull/3021/commits/2699376b07180ec4cdfbaeb14062abb4353fab00

Reply:
well it isn't present in the PR?

Reply:
Sorry I didn't get you

Reply:
but if you are asking for the PR ID   the ID is 3021 
  
  https://github.com/cpinitiative/usaco-guide/pull/3021 
  
  the link that I've shared in my previous comment redirects to view the new changes made by me which is in the above PR Mentioned